orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -51,12 +52,11 @@ def _forward(self, *inputs, **kwargs):
         register_custom_ops_pytorch_exporter.register_custom_op(is_ortmodule=True)
 
         # User module is wrapped to use its initializers and save computed gradients
-        self._original_module = module
+        # along with the module that flattens both input and output of the user module
+        # inside ModuleMetadata
+        self._module = ModuleMetadata(module, _io._FlattenedModule(module))
```

Comment:
`self._module` does not reflect what it stores (`ModuleMetadata`)
```suggestion
        self._module_info = _PytorchModuleMetadata(module, _io._FlattenedModule(module))
```
Using the _ before private class